### DISCUSSION TITLE
New - Add RGB color variant

### DISCUSSION THREAD

**Discussion by RazvanCoada** [Legal-and-General/canopy#913] [Canopy requests]
In order to use colors with transparency we need the RGB format to use with css rgba().
Similar to how we added --color-super-blue-rgb we need the following colors: 

--color-charcoal-rgb
--color-leafy-green-rgb
--color-dandelion-yellow-rgb

**Comment by elenagarrone** [comment]
@RazvanCoada do you know what components would need these colours? do you have designs?

**Reply by RazvanCoada** [reply]
The component that will use them is not part of canopy at the moment, but here are the designs:
https://legalandgeneral.invisionapp.com/console/share/8PG6TZSCQ7J/922373559

**Reply by elenagarrone** [reply]
Sounds ok to me to add them, @alexcanning any opinion on your side?

**Reply by alexcanning** [reply]
Sounds good to me!

**Comment by elenagarrone** [comment]
@RazvanCoada would you be able to create a PR for this?

**Reply by RazvanCoada** [reply]
Yes, I will add one today. Thank you

**Comment by elenagarrone** [comment]
PR merged: https://github.com/Legal-and-General/canopy/pull/920
